packages/cqrs/lib/src/transport_types.dart

```diff
@@ -23,16 +23,16 @@ abstract class CqrsMethod {
 }
 
 /// Query describing a criteria for a query and the results it returns.
-abstract class Query<T> implements CqrsMethod {
+abstract interface class Query<T> implements CqrsMethod {
```

Comment:
can this interface extend that interface? I'm not sure whether it makes any real difference, but interfaces usually extend other interfaces 🤔

Reply:
Changed, I don't think it will make any difference. But you are right, it makes more sense to extend.